### DISCUSSION TITLE
After Installing, and everything is exposed in /admin/ templates cannot be found

### DISCUSSION THREAD

**Discussion by michaeloliveraz** [barseghyanartur/django-fobi#280] [Q&A] [answered]
Fobi is installed in mysite along with djangorestframework and the dependencies.  creating forms and form elements works fine in admin, but when I try to view the form I get...

<html>
<body>
<h1 style="scroll-behavior: auto !important; padding: 0px; margin: 0px; font-weight: normal; color: rgb(0, 0, 0); font-family: sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">AttributeError at /fobi/view/ringotingo/</h1><pre class="exception_value" style="scroll-behavior: auto !important; padding: 0px; margin: 10px 0px; font-size: 1.5em; white-space: pre-wrap; word-break: break-word; font-family: sans-serif; color: rgb(87, 87, 87); font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">'FormHandlerPluginDataStorage' object has no attribute 'to_email'</pre>

Request Method: | POST
-- | --
http://localhost:8000/fobi/view/ringotingo/
4.1.1
AttributeError
'FormHandlerPluginDataStorage' object has no attribute 'to_email'
C:\Users\oliver\base\correlator2v2\correlator2home\venv\lib\site-packages\fobi\contrib\plugins\form_handlers\mail\mixins.py, line 80, in send_email
fobi.views.function_based.view_form_entry
C:\Users\oliver\base\correlator2v2\correlator2home\\scripts\python.exe
3.10.6
['C:\\Users\\oliver\\base\\correlator2v2\\correlator2home',  'C:\\Python3\\python310.zip',  'C:\\Python3\\DLLs',  'C:\\Python3\\lib',  'C:\\Python3',  'C:\\Users\\oliver\\base\\correlator2v2\\correlator2home\\venv',  'C:\\Users\\oliver\\base\\correlator2v2\\correlator2home\\venv\\lib\\site-packages']
Sat, 10 Sep 2022 03:55:21 +0000

</body>
</html>

my settings INSTALLED_APS

```
    'fobi.contrib.plugins.form_handlers.mail',
    'fobi.contrib.plugins.form_handlers.mail_sender', 
    'fobi.contrib.apps.drf_integration.form_handlers.mail',

```

**Comment by barseghyanartur** [comment]
It's probably, you created forms not using the django-fobi management interface, but using Django admin.

**Reply by michaeloliveraz** [reply]
Ok, that helps and yes I used the Django Admin interface, however I have another post on an error when trying to add an Text element in what I assume is the fobi management.  Shoudn't they both work? Your help is appreciated.

**Reply by michaeloliveraz** [reply]
Ok, I added the Fobi Management URLs to my home index.html and will use those instead of Django Admin pages. At some point, I may disable the admin pages if I cannot put the Fobi Management Lins in the Django Admin Pages in place of the auto-generated ones.

**Comment by michaeloliveraz** [comment, marked as answer]
Use the Fobi Management pages like http://localhost:8000/fobi/ for the dashboard.
